CHANGELOG.md

```diff
@@ -5,32 +5,38 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [2.1.0] - 2019-06-11
-The main focus of this minor release is cleaning up the build system and extracting required compiled libraries from the HELICS build process, no changes in the C api, and few additionals and deprecations in the C++ API related to command line arguments.  
+The main focus of this minor release is cleaning up the build system and extracting required compiled libraries from the HELICS build process, no changes in the C API, and a few additions and deprecations in the C++ API related to command line arguments.  
 ### Changed
    - remove use of boost::program options and replace usage with CLI11
    - remove boost::asio and replace with a submodule for ASIO
    - remove included fmt code and replace with submodule
-   - remove jsoncpp code and replace with a submodule which generates a compiled library- this removed the need to continually regenerate the single header/file with customized namespaces, though if you are manually including helics the HELICS::jsoncpp needs to be added as a library.
+   - remove JsonCpp code and replace with a submodule which generates a compiled library - this removed the need to continually regenerate the single header/file with customized namespaces, though if you are using the helics-static library built with a separate JsonCpp static library, the HELICS copy of the jsoncpp static library must be linked with manually (for build systems other than CMake such as waf, meson, premake, etc).  Also included is an option to incorporate JsonCpp as an object library within a single helics-static library (default on macOS/Linux), and create a target HELICS::jsoncpp_headers.
    - extract several containers used in HELICS to a separate repository for better maintenance and possible reuse elsewhere.  Any reference to the containers library was removed from the Public API.
    - all required references to boost were removed from the public API.  
    - the logger headers were split into two sections.  The logger.h which includes the logger objects for use in federates was split from the loggerCore which is not publicly accessible.  
    - The command line arguments are error checked and the help prints all available options (thanks to CLI11)
    - the core tests and common tests now use google test instead of boost test.  More tests are expected to be migrated in the future.  
+   - updates to the HELICSConfig.cmake file that gets installed to be more resilient to different directory structures. 
+   - Use ZMQ as a subproject if needed instead of an autobuild and install it as a target if needed.  
```

Comment:
```suggestion
   - use ZMQ as a subproject if needed instead of an autobuild and install it as a target if needed. The CMake option to enable this is ZMQ_LOCAL_BUILD, replacing AUTOBUILD_ZMQ.
```